Makefile

```diff
@@ -136,6 +144,20 @@ auto_usera.o: \
 compile auto_usera.c
 	./compile auto_usera.c
 
+auto_usergroupnames.h: \
+auto-str conf-users conf-groups
+	( ./auto-str auto_usera `head -1 conf-users` \
```

Comment:
This should be "tail -n 1" for extended Posix compatibility.

Reply:
Hmmm, I don't see `tail -n` anywhere else in `Makefile`. Pretty sure I copy-pasted the form followed elsewhere in the file. If this is an important family of changes to make (maybe because it fixes broken builds on some platform?), please propose it as a separate PR.

Reply:
If things are that way in other places I'm ok with it for now.